docs/src/developing/programming-model/accounts.md

```diff
@@ -24,9 +24,9 @@ uses an _address_ to look up an account. The address is a 256-bit public key.
 ## Signers
 
 Transactions may include digital [signatures](terminology.md#signature)
-corresponding to the accounts' public keys referenced by the transaction. Such
+corresponding to the account's public keys referenced by the transaction. Such
```

Comment:
This change is actually a bit wrong, since accounts (and signatures) can be plural. How about we write around it? Singluar/plural isn't handled well in the paragraph anyway, and I think the terminology is not very clear.

Maybe:
```
Transactions include one or more digital [signatures](terminology.md#signature)
each corresponding to an account address referenced by the transaction. Each of these
addresses must be the public key of an ed25519 keypair, and the signature signifies
that the holder of the matching private key signed, and thus, "authorized" the transaction.